lib/Interpreter/Compatibility.h

```diff
@@ -247,6 +275,34 @@ inline llvm::Error Undo(clang::Interpreter& I, unsigned N = 1) {
 #endif
 }
 
+inline void codeComplete(clang::Interpreter& I, const char* code, std::vector<std::string>& Results) {
+#if CLANG_VERSION_MAJOR >= 18
+  auto CB = clang::IncrementalCompilerBuilder();
+  auto CI = CB.CreateCpp();
```

Comment:
We should add a fixme note here saying that we should match the invocation arguments of the main interpreter. That can affect the returned completion results.